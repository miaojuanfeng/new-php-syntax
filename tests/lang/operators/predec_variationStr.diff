002+ UNKNOWN:0
002- int(-1)
004+ UNKNOWN:0
004- int(64)
006+ UNKNOWN:0
006- int(-45)
008+ UNKNOWN:0
008- float(0.2)
010+ UNKNOWN:0
010- float(-8.7)
012+ UNKNOWN:0
012- string(3) "abc"
014+ UNKNOWN:0
014- string(6) "123abc"
016+ UNKNOWN:0
016- float(12299999)
018+ UNKNOWN:0
018- string(8) "123e5xyz"
020+ UNKNOWN:0
020- string(7) " 123abc"
022+ UNKNOWN:0
022- string(7) "123 abc"
024+ UNKNOWN:0
024- string(7) "123abc "
026+ UNKNOWN:0
026- string(4) "3.4a"
028+ UNKNOWN:0
028- string(4) "a5.9"
030+ [Wed Nov 18 22:29:54 2015]  Script:  '/home/mjf/php/tests/lang/operators/predec_variationStr.php'
031+ /home/mjf/php/Zend/zend_vm_execute.h(30538) :  Freeing 0x2AB6DA6C6EF0 (32 bytes), script=/home/mjf/php/tests/lang/operators/predec_variationStr.php
032+ Last leak repeated 13 times
033+ === Total 14 memory leaks detected ===