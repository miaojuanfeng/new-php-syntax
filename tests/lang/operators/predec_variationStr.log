
---- EXPECTED OUTPUT
--- testing: '0' ---
int(-1)
--- testing: '65' ---
int(64)
--- testing: '-44' ---
int(-45)
--- testing: '1.2' ---
float(0.2)
--- testing: '-7.7' ---
float(-8.7)
--- testing: 'abc' ---
string(3) "abc"
--- testing: '123abc' ---
string(6) "123abc"
--- testing: '123e5' ---
float(12299999)
--- testing: '123e5xyz' ---
string(8) "123e5xyz"
--- testing: ' 123abc' ---
string(7) " 123abc"
--- testing: '123 abc' ---
string(7) "123 abc"
--- testing: '123abc ' ---
string(7) "123abc "
--- testing: '3.4a' ---
string(4) "3.4a"
--- testing: 'a5.9' ---
string(4) "a5.9"
===DONE===
---- ACTUAL OUTPUT
--- testing: '0' ---
UNKNOWN:0
--- testing: '65' ---
UNKNOWN:0
--- testing: '-44' ---
UNKNOWN:0
--- testing: '1.2' ---
UNKNOWN:0
--- testing: '-7.7' ---
UNKNOWN:0
--- testing: 'abc' ---
UNKNOWN:0
--- testing: '123abc' ---
UNKNOWN:0
--- testing: '123e5' ---
UNKNOWN:0
--- testing: '123e5xyz' ---
UNKNOWN:0
--- testing: ' 123abc' ---
UNKNOWN:0
--- testing: '123 abc' ---
UNKNOWN:0
--- testing: '123abc ' ---
UNKNOWN:0
--- testing: '3.4a' ---
UNKNOWN:0
--- testing: 'a5.9' ---
UNKNOWN:0
===DONE===
[Wed Nov 18 22:29:54 2015]  Script:  '/home/mjf/php/tests/lang/operators/predec_variationStr.php'
/home/mjf/php/Zend/zend_vm_execute.h(30538) :  Freeing 0x2AB6DA6C6EF0 (32 bytes), script=/home/mjf/php/tests/lang/operators/predec_variationStr.php
Last leak repeated 13 times
=== Total 14 memory leaks detected ===
---- FAILED
