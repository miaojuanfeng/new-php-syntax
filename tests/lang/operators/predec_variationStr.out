--- testing: '0' ---
UNKNOWN:0
--- testing: '65' ---
UNKNOWN:0
--- testing: '-44' ---
UNKNOWN:0
--- testing: '1.2' ---
UNKNOWN:0
--- testing: '-7.7' ---
UNKNOWN:0
--- testing: 'abc' ---
UNKNOWN:0
--- testing: '123abc' ---
UNKNOWN:0
--- testing: '123e5' ---
UNKNOWN:0
--- testing: '123e5xyz' ---
UNKNOWN:0
--- testing: ' 123abc' ---
UNKNOWN:0
--- testing: '123 abc' ---
UNKNOWN:0
--- testing: '123abc ' ---
UNKNOWN:0
--- testing: '3.4a' ---
UNKNOWN:0
--- testing: 'a5.9' ---
UNKNOWN:0
===DONE===
[Wed Nov 18 22:29:54 2015]  Script:  '/home/mjf/php/tests/lang/operators/predec_variationStr.php'
/home/mjf/php/Zend/zend_vm_execute.h(30538) :  Freeing 0x2AB6DA6C6EF0 (32 bytes), script=/home/mjf/php/tests/lang/operators/predec_variationStr.php
Last leak repeated 13 times
=== Total 14 memory leaks detected ===